oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -322,6 +323,19 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
             }
         }
 
+        if (!client.getIsActive()) {
+            String errorMsg = "Client configured as not active in Client Registry";
```

Comment:
this makes sense as a log message, but I think less so as an RP facing error message. "Client Registry" won't mean anything to them. For the error response we could go with something like `client disabled`

Reply:
Gabe originally had something like this and I suggested he changed it :)
https://github.com/govuk-one-login/authentication-api/pull/4979#discussion_r1698342819